tests/masonry/builders/fixtures/with_readme_files/pyproject.toml

```diff
@@ -0,0 +1,19 @@
+[tool.poetry]
```

Comment:
I'd rather share/re-use these fixtures (and have one single, complete version) rather than duplicate two partial versions.